Added OSINT Rizzler

REGEX777 opened the pull request:
# TerminalCraft Project Submission

## Project Details

1. **What is the name of your project?**
OSINT Rizzler
2. **How many hours did you spend developing this?**
   _(Give your best estimate or show proof using hackatime/wakatime)_
4.5
3. **Briefly describe what your project is. What makes it unique?**
   _(Provide a brief description of the functionality or innovation)_
OSINT Rizzler is a powerful reconnaissance tool designed to help gather publicaly available information about domains, IP addresses, emails, and websites. This tool provides various OSINT (Open Source Intelligence) techniques to facilitate information gathering.

4. **How many people have tested your extension?**
   _(We recommend at least 3)_
3
5. **On What operating systems have you tested this project?**
   _(MacOS/Linux/Windows)_
Linux/Windows
6. **Does your project include a Readme file with instructions on how to use it?**
   _(Yes/No)_
Yes
---

## Additional Comments

(Feel free to provide any additional context or notes here)

JosiasAurel reviewed (changes requested):
Pls fix this.
<img width="1466" alt="image" src="https://github.com/user-attachments/assets/fa8c48b0-d78c-4928-a09a-f538c1f77f12" />

Also 4 hours is below time requirement

REGEX777 commented:
> Pls fix this. <img alt="image" width="1466" src="https://private-user-images.githubusercontent.com/47951376/425010146-fa8c48b0-d78c-4928-a09a-f538c1f77f12.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3NDI0ODA1MDIsIm5iZiI6MTc0MjQ4MDIwMiwicGF0aCI6Ii80Nzk1MTM3Ni80MjUwMTAxNDYtZmE4YzQ4YjAtZDc4Yy00OTI4LWEwOWEtZjUzOGMxZjc3ZjEyLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNTAzMjAlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjUwMzIwVDE0MTY0MlomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWJhMDcyNzdiZjgzOTNjMGQwN2QwMjUzNjgxN2YwMDBjZjAyOTMyMjU5MzBhZDViZDdkMjdhOGIyZTcwMjFjYjcmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.h1zGzAkFhsk0lvABn0MnrQqdRwX3PtFQW-APMK48l9Y">
> 
> Also 4 hours is below time requirement

I have updated the repo and fixed the issues.

JosiasAurel commented:
still not fixed

REGEX777 commented:
Are you still getting the issue with whois? Cus I did fix the issue, it was related to the requirements.txt, also could you share a screenshot?
